libcontainer/specconv/spec_linux.go

```diff
@@ -1127,6 +1175,53 @@ func parseMountOptions(options []string) *configs.Mount {
 	return &m
 }
 
+// parseListSet parses "list set" syntax ("0,61-63,2") into a list ([0, 61, 62, 63, 2]).
+func parseListSet(listSet string, minValue, maxValue int) ([]int, error) {
```

Comment:
Needs a unit test

Reply:
Thanks @AkihiroSuda! Added a test and removed unnecessary `if` from the parser.